lib/Controller/ChatController.php

```diff
@@ -592,7 +592,43 @@ public function deleteMessage(int $messageId): DataResponse {
 
 		$bridge = $this->matterbridgeManager->getBridgeOfRoom($this->room);
 
-		$response = new DataResponse($data, $bridge['enabled'] ? Http::STATUS_ACCEPTED: Http::STATUS_OK);
+		$response = new DataResponse($data, $bridge['enabled'] ? Http::STATUS_ACCEPTED : Http::STATUS_OK);
+		if ($this->participant->getAttendee()->getReadPrivacy() === Participant::PRIVACY_PUBLIC) {
```

Comment:
I'm guessing this is an additional fix not directly related to clearing the chat history ?

if yes you might want to make that a separate PR/commit so we can backport, if applicable

Reply:
Yeah we could/should. The header is documented, but the clients are forgiving
I will manually backport it together with the `deleteMessages` fix and the broken integration test